notifier.go

```diff
@@ -23,7 +23,6 @@ const userAgent = notifierName + "/" + notifierVersion
 const waitTimeout = 5 * time.Second
 
 const httpEnhanceYourCalm = 420
```

Comment:
What about this one?

Reply:
This is a user defined status code. To read more:
[evertpot.com/http/420-enhance-your-calm](https://evertpot.com/http/420-enhance-your-calm)
It is not defined by [pkg.go.dev/net/http#pkg-constants](https://pkg.go.dev/net/http#pkg-constants)